<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>979</width>
    <height>724</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>隶书</family>
      <pointsize>23</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 0);</string>
    </property>
    <property name="text">
     <string>IM即时通</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="IPTableWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>211</width>
      <height>611</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="CurrentLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 97, 100);</string>
    </property>
    <property name="text">
     <string>192.168.123.123</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="RxTextEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>140</y>
      <width>671</width>
      <height>391</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTextEdit" name="TxTextEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>560</y>
      <width>621</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="MessPushButton">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>567</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>发  送</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FilePushButton">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>640</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>文  件</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
